src/TestSuppressions.cs

```diff
@@ -0,0 +1,3 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Reliability", "CA2007:Consider calling ConfigureAwait on the awaited task", Justification = "Test file")]
```

Comment:
I think this should really say "test project" to match our other suppressions, but let's not go through CI all over again just for that.